src/app/core/navbar/navbar.component.css

```diff
@@ -118,3 +118,27 @@
   color: red;
   font-weight: bold;
 }
+
+.login-btn-container {
+  display: none;
+}
+.login-btn {
+  background-color: transparent;
+  border-width: 3px;
```

Comment:
Hey @kishanmanoharan we do all of our sizing at Virtrolio using `em` to help with scaling. Could you fix this? You can use an `em` to `px` converter like this one: https://www.w3schools.com/tags/ref_pxtoemconversion.asp

Reply:
Border widths don't need to be in `em`.

Reply:
I will change and push that

Reply:
No, use pixels. @DabeerAzeez we don't need to use `em` for borders, that doesn't make sense

Reply:
Why not? I thought we wanted our whole website to scale properly? @ricetech